jobs/mongodb_migration/src/mongodb_migration/collector.py

```diff
@@ -32,4 +38,12 @@ def get_migrations(self) -> List[Migration]:
                 version="20230126164900",
                 description="add 'priority' field to jobs in queue database",
             ),
+            MigrationCacheUpdateSplitNames(
+                version="20230216112500",
+                description="update 'kind' field in cache from /split-names to /split-names-streaming",
```

Comment:
shouldn't it be `/split-names-from-streaming`?

Reply:
Thanks, I will send a new PR to fix this

Reply:
https://github.com/huggingface/datasets-server/pull/837/files